### ISSUE TITLE
Comments For Last Homework

### ISSUE THREAD

**Issue by ztd5049** [DigitalMitford/DM_Journal_ste#6]
Hello! I really enjoyed the layout of the website. I think it flows nicely as you're going from page to page. The quick links on the side of the letters pages are super useful for navigation. The overall website CSS looks good and the theming of the each page feels complete and tactile

**Comment by gzc5211** [comment]
I like your guys' web outline. and the about page. It tells a very clear message of what I'm looking for. also, I think you guys did a great job of showing the different time periods. I can just pick the date I want to go. Good Job!!!!!!

**Comment by spookybeetle** [comment]
The aesthetic of the first page is lovely! It is very pleasing on the eyes. I would maybe try to make the color pallets match across each page? It was very jarring to go from the landing page to the following pages because the color pallet changed drastically.

**Comment by SavannahRicks** [comment]
@ebeshero We declare this project to be finished and ready to be evaluated!

**Comment by natalyamyers** [comment]
I agree with the other comments, the color palette on some pages could use some tweaking. Generally, for background shapes and colors, such as the brown you have for the 1819 and 1820 pages, it works well to do something in a lower opacity or a more muted tone. This way, a viewers focus is on the content of the page, not what's going on in the background.
